### PR TITLE
doc: installation: review section

### PR DESCRIPTION
**Pull request by greg-fer**
Edited Installation pages for latest updates in toolchain management. NCSDK-22323.

----

- [x] Add redirects for the removed pages.
- [x] Implement feedback from the meeting with stakeholders.

----

Goals with this rework:
- Have one page for the installation procedures for both VSC extension and command line, regardless of manual or automatic installation

Reasons:
- We have now 5 different approach points for starting work with the NCS and installing the requirements: TCM, Quick Start, VSC, nRF Util toolchain-manager for command line, manual command line steps
- Automatic procedure still required some manual steps, so the division automatic/manual was not accurate (and was going mostly along the lines of using Nordic tools or not)
- With so many approach points, maintaining two different pages about installing makes it really hard and time-consuming to include all these approaches in a clear and user-friendly manner
- We will be removing some installation docs for VSC extension in long term, so we need to unify the content about VSC

### CODE REVIEW

File: doc/nrf/installation/install_ncs.rst [hunk 1]
```diff
@@ -0,0 +1,375 @@
...
+         If you have received a custom URL or a custom bundle ID for installing toolchain, you can use the following set of commands to provide it, replacing the respective parameters:
+
+         .. parsed-literal::
+            :class: highlight
+
+            nrfutil toolchain-manager config --show
+            nrfutil toolchain-manager config --set toolchain-index=*custom_toolchain_URL*
+            nrfutil toolchain-manager install --bundle-id *custom_bundle_ID*
+
+         To read more about these commands, use the ``nrfutil toolchain-manager help`` command to open the help page.
```
[the remark this answers is not in the document]
Comment (greg-fer) [reply doc/nrf/installation/install_ncs.rst:130]: > only cover things such as completion

For now... ;)

Will use --help for the time being.

File: doc/nrf/installation/install_ncs.rst [hunk 2]
```diff
@@ -0,0 +1,375 @@
...
+         The list of available SDK versions appears in the |VSC|'s quick pick.
+      #. Select the SDK version to install.
+         In most cases, you want the SDK version that matches your toolchain version.
+         The SDK installation starts and it can take several minutes.
+
+   .. group-tab:: Command line
+
+      To clone the repositories, complete the following steps:
+
+      1. Create a folder named :file:`ncs`.
```
[the remark this answers is not in the document]
Comment (greg-fer) [reply doc/nrf/installation/install_ncs.rst:184]: Good point!

File: doc/nrf/installation/updating.rst [hunk 3]
```diff
@@ -17,82 +17,78 @@ However, if you work with a specific release of the |NCS|, you do not need to up
...
-Running ``west update`` updates the project repositories to the state specified in this manifest file.
+            .. parsed-literal::
+               :class: highlight
 
-.. include:: installing.rst
-   :start-after: west-error-start
-   :end-before: west-error-end
+               git fetch origin
+               git checkout |release|
+               west update
```
Comment (nordicjm) [inline doc/nrf/installation/updating.rst:61]: This seems like it should be in the part earlier where I left a comment

File: doc/nrf/installation/install_ncs.rst [hunk 4]
```diff
@@ -0,0 +1,386 @@
...
+         This is one level up from the location where you installed the toolchain.
+         This directory will hold all |NCS| repositories.
+
+      #. Determine the identifier of the revision of the |NCS| you want to work with.
+         See the table below for more information.
+
+      #. On the command line, go to the :file:`ncs` directory (``cd ncs``) and initialize west with the revision of the |NCS| that you want to check out, replacing *nRFConnectSDK_revision* with the identifier:
+
+         .. parsed-literal::
+            :class: highlight
```
Comment (nordicjm) [inline doc/nrf/installation/install_ncs.rst:203]: isn't this just a full duplicate of the same information as the other commit has in installing.rst?

I see the file has moved, there should probably be a file redirect added from old to new URL

File: doc/nrf/installation/recommended_versions.rst [hunk 5]
```diff
@@ -77,15 +77,13 @@ The table uses the following tier definitions to categorize the level of operati
...
+  There are some Zephyr features that are currently only available on Linux, including:
 
-There are some Zephyr features that are currently only available on Linux, including:
-
-* twister
-* BlueZ integration
-* net-tools integration
-* Native Port (native_posix)
-* BabbleSim
+  * twister
```
Comment (nordicjm) [inline doc/nrf/installation/recommended_versions.rst:82]: @PerMac

File: doc/nrf/installation/recommended_versions.rst [hunk 6]
```diff
@@ -71,38 +55,59 @@ The following table shows the operating system versions that support the |NCS| t
...
-   The |NCS| tools are not supported by the older versions of the operating system.
+   Not applicable
+     The specified architecture is not supported for the respective operating system.
+
+Linux-only Zephyr features
+==========================
+
+There are some Zephyr features that are currently only available on Linux, including:
+
+* twister
```
[the remark this answers is not in the document]
Comment (greg-fer) [reply doc/nrf/installation/recommended_versions.rst]: This is a very old statement, so I'd be happy if you can look at the whole list there, @PerMac

File: doc/nrf/installation.rst [hunk 7]
```diff
@@ -4,29 +4,29 @@
...
 
-The |NCS| is a combination of software developed by Nordic Semiconductor and open source projects, hosted as :ref:`ncs_git_intro` repositories in the `nrfconnect GitHub organization`_.
-These repositories and their versions form the SDK's code base and are managed using Zephyr's :ref:`ncs_west_intro` tool.
+To set up the |NCS|, use one of the following installation methods:
 
-When you install the |NCS|, you set up the following SDK structure:
+* :ref:`Install the nRF Connect SDK automatically using nRF Connect for Desktop <auto_installation>`.
+  This will install all the :ref:`requirements <requirements>`, including the |VSC| and the `nRF Connect for Visual Studio Code`_ extension, which is recommended for creating and building applications.
+* :ref:`Install the nRF Connect SDK toolchain manually <manual_installation>`.
+  This is an option if you have any issues when installing the |NCS| automatically.
```
[the remark this answers is not in the document]
Comment (greg-fer) [reply doc/nrf/installation.rst]: This comes from the first commit and was later reworked. Apologies for confusion, I've merged the two commits now.

File: doc/nrf/installation/installing.rst [hunk 8]
```diff
@@ -214,6 +204,11 @@ Install west
 ************
 
 To manage the combination of repositories and versions, the |NCS| uses :ref:`Zephyr's west <zephyr:west>`.
+When you install manually, you install west separately from other :ref:`required tools <requirements_toolchain_tools>`.
+
+.. note::
+   You only need to install west once.
+   You do not have to complete this step if you are installing a new version of the |NCS|.
```
[the remark this answers is not in the document]
Comment (greg-fer) [reply doc/nrf/installation/installing.rst]: This comes from the first commit and was later reworked. Apologies for confusion, I've merged the two commits now.

File: doc/nrf/installation/installing.rst [hunk 9]
```diff
@@ -294,87 +295,94 @@ To clone the repositories, complete the following steps:
...
+     .. code-block:: console
 
-     west init -m https://github.com/nrfconnect/sdk-nrf --mr main
+        west init -m https://github.com/nrfconnect/sdk-nrf --mr main
 
    This will clone the manifest repository `sdk-nrf`_ into :file:`nrf`.
 
-#. Enter the following command to clone the project repositories::
+   Initializing west with a specific revision of the manifest file does not lock your repositories to this version.
+   Checking out a different branch or tag in the `sdk-nrf`_ repository and running ``west update`` changes the version of the |NCS| that you work with.
```
[the remark this answers is not in the document]
Comment (greg-fer) [reply doc/nrf/installation/installing.rst]: Fixed.

File: doc/nrf/installation/install_ncs.rst (hunk 4 above)
Comment (greg-fer) [reply doc/nrf/installation/install_ncs.rst:203]: Apologies for confusion, I've merged the two commits now.

File: doc/nrf/installation/updating.rst (hunk 3 above)
Comment (greg-fer) [reply doc/nrf/installation/updating.rst:61]: Tried to rephrase the headers for clarity.

File: doc/nrf/installation/recommended_versions.rst (hunk 5 above)
Comment (PerMac) [reply doc/nrf/installation/recommended_versions.rst:82]: Twister is available on Windows and works.  Tests for native posix and qemu won't work, but it is not a twister issue, but the fact that does platforms are not supported  on Windows

Comment (PerMac) [reply doc/nrf/installation/recommended_versions.rst:82]: BlueZ and net-tools: no idea, 
no native posix on Windows -> correct
babblesim is based on posix -> correct

Comment (PerMac) [reply doc/nrf/installation/recommended_versions.rst]: As for mac:
twister in general works
In particular qemu and hw should work
native posix doesn't work, hence bsim as well

File: doc/nrf/installation/install_ncs.rst [hunk 10]
```diff
@@ -0,0 +1,386 @@
...
+      #. Select the toolchain version to install.
+         The toolchain version should match the |NCS| version you are going to work with.
+         If you have received a custom URL for installing toolchain, you can provide it using the :guilabel:`Change Toolchain Index` button in the quick pick's header.
+         The toolchain installation starts in the background, as can be seen in the notification that appears.
+
+      When you install the toolchain for the first time, the installed version is automatically selected for your project.
+
+   .. group-tab:: Command line
+
+      1. Open a command prompt window.
```
Comment (skogseth) [inline doc/nrf/installation/install_ncs.rst:93]: I think "command prompt" is a Windows specific term, "terminal" is a more universal term (also, nrfutil doesn't officially support command prompt, only `bash`, `zsh` & `pwsh`, so it's extra unfortunate)

File: doc/nrf/installation/install_ncs.rst [hunk 11]
```diff
@@ -0,0 +1,386 @@
...
+
+         For example:
+
+         .. code-block:: console
+
+            nrfutil toolchain-manager install --ncs-version v2.0.0
+
+         The toolchain version should match the |NCS| version you are going to work with.
+
+      The toolchain is installed by default at :file:`C:/ncs/toolchains` on Windows, :file:`~/ncs` on Linux, and :file:`/opt/nordic/ncs` on macOS.
```
Comment (skogseth) [inline doc/nrf/installation/install_ncs.rst:122]:
```suggestion
      The toolchain is installed by default at :file:`C:\ncs` on Windows, :file:`~/ncs` on Linux, and :file:`/opt/nordic/ncs` on macOS.
```
Alternatively add the "toolchains" directory to all the paths

Comment (greg-fer) [reply doc/nrf/installation/install_ncs.rst:122]: I'll keep forward slash, as it is also supported on Windows.

Comment (nordicjm) [reply doc/nrf/installation/install_ncs.rst]: On windows in command prompt? Are you sure?

Comment (greg-fer) [reply doc/nrf/installation/install_ncs.rst]: Yes

**Review by ihansse — APPROVED** [review]
Looks good to me!

File: doc/nrf/installation/updating.rst [hunk 12]
```diff
@@ -17,82 +17,79 @@ However, if you work with a specific release of the |NCS|, you do not need to up
...
+      **Example: Switching to a revision (SHA, branch, or tag)**
 
-.. parsed-literal::
-   :class: highlight
+         .. toggle::
 
-   git fetch origin
-   git checkout |release|
-   west update
+            To update to a particular revision, make sure that you have that particular revision on your local file system before you check it out (by running ``git fetch origin``):
```
Comment (DeidreCasey) [inline doc/nrf/installation/updating.rst:72]:
```suggestion
            To update to a particular revision, make sure that you have that particular revision on your local file system before you check it out by running ``git fetch origin``:
```

I don't think parentheses are needed here.

**Review by DeidreCasey — CHANGES_REQUESTED** [review]

**Comment by DeidreCasey** [comment]
@carlescufi What is your thinking behind wanting to keep them separate? Having them combined in this current state allows for information to be more flexible when we add in new features like workflow maps to Zoomin.

Additionally, the idea of an "advanced user" is a subjective term, so what criteria are you thinking of that would make them "advanced"?

**Review by skogseth — CHANGES_REQUESTED** [review]

File: doc/nrf/installation/recommended_versions.rst [hunk 13]
```diff
@@ -69,40 +53,58 @@ The following table shows the operating system versions that support the |NCS| t
...
 
 |NCS| toolchain
 ***************
 
 The |NCS| :term:`toolchain` includes the Zephyr SDK and then adds on top of it tools and modules required to build |NCS| samples and applications.
 
+.. note::
+
+   Before you start setting up the |NCS| toolchain, install available updates for your operating system.
+   The |NCS| tools are not supported by the older versions of the operating system.
```
[the remark this answers is not in the document]
Comment (greg-fer) [reply doc/nrf/installation/recommended_versions.rst:98]: @kenr , @kotlarek-nordic , could you answer @DeidreCasey 's question? The line was originally introduced in #8137 with your input.

**Review by FrancescoSer — APPROVED** [review]

File: doc/nrf/installation/install_ncs.rst [hunk 14]
```diff
@@ -0,0 +1,599 @@
...
+   .. group-tab:: nRF Connect for Visual Studio Code
+
+      1. Open the nRF Connect extension in |VSC| by clicking its icon in the :guilabel:`Activity Bar`.
+      #. In the extension's :guilabel:`Welcome View`, click on :guilabel:`Manage toolchains`.
+         The list of actions appears in the |VSC|'s quick pick.
+      #. Click :guilabel:`Install Toolchain`.
+         The list of available toolchain versions appears in the |VSC|'s quick pick.
+      #. Select the toolchain version to install.
+         The toolchain version should match the |NCS| version you are going to work with.
+         If you have received a custom URL for installing toolchain, you can provide it using the :guilabel:`Change Toolchain Index` button in the quick pick's header.
```
Comment (DeidreCasey) [inline doc/nrf/installation/install_ncs.rst:82]:
```suggestion
         If you have received a custom URL for installing the toolchain, you can provide it using the :guilabel:`Change Toolchain Index` button in the quick pick's header.
```

File: doc/nrf/installation/install_ncs.rst [hunk 15]
```diff
@@ -0,0 +1,599 @@
...
+
+         .. code-block:: console
+
+            nrfutil toolchain-manager install --ncs-version v2.0.0
+
+         The toolchain version should match the |NCS| version you are going to work with.
+
+      The toolchain is installed by default at :file:`C:/ncs/toolchains` on Windows, :file:`~/ncs/toolchains` on Linux, and :file:`/opt/nordic/ncs/toolchains` on macOS.
+
+      If you have received a custom URL for installing toolchain, you can use the following command to set it as default, replacing the respective parameters:
```
Comment (DeidreCasey) [inline doc/nrf/installation/install_ncs.rst:120]:
```suggestion
      If you have received a custom URL for installing the toolchain, you can use the following command to set it as default, replacing the respective parameters:
```

**Review by DeidreCasey — CHANGES_REQUESTED** [review]

File: doc/nrf/installation/install_ncs.rst [hunk 16]
```diff
@@ -0,0 +1,599 @@
...
+
+                  nrfutil toolchain-manager launch --terminal
+
+            .. tab:: Linux
+
+               .. code-block:: console
+
+                  nrfutil toolchain-manager launch --shell
+
+            .. tab:: Windows
```
Comment (nordicjm) [inline doc/nrf/installation/install_ncs.rst:209]: duplicate windows?

File: doc/nrf/installation/install_ncs.rst [hunk 17]
```diff
@@ -0,0 +1,599 @@
...
+
+            nrfutil toolchain-manager install --ncs-version *toolchain_version*
+
+         For example:
+
+         .. code-block:: console
+
+            nrfutil toolchain-manager install --ncs-version v2.0.0
+
+         The toolchain version should match the |NCS| version you are going to work with.
```
Comment (nordicjm) [inline doc/nrf/installation/install_ncs.rst:116]: Incorrect

File: doc/nrf/installation/install_ncs.rst [hunk 18]
```diff
@@ -0,0 +1,599 @@
...
+In addition to the steps mentioned above, if you want to build and program your application from the command line, you have to set up your command-line build environment by defining the required environment variables every time you open a new command-line or terminal window.
+See :ref:`zephyr:important-build-vars` for more information about the various relevant environment variables.
+
+Define the required environment variables as follows, depending on your operating system:
+
+.. tabs::
+
+   .. tab:: Windows
+
+      Navigate to the :file:`ncs` directory and enter the following command: ``zephyr\zephyr-env.cmd``
```
Comment (nordicjm) [inline doc/nrf/installation/install_ncs.rst:311]: This should state using what, because someone reading this is going to open a file explorer view, go to this file, double click it and think what's that meant to do

File: doc/nrf/installation/install_ncs.rst [hunk 19]
```diff
@@ -0,0 +1,599 @@
...
+                  python -m venv ncs\.venv
+
+            #. Activate the virtual environment:
+
+               .. code-block:: bash
+
+                  :: cmd.exe
+                  ncs\.venv\Scripts\activate.bat
+                  :: PowerShell
+                  ncs\.venv\Scripts\Activate.ps1
```
Comment (nordicjm) [inline doc/nrf/installation/install_ncs.rst:427]: Let's get rid of powershell unless someone can confirm it is supported and works (I have heard of no such support)

File: doc/nrf/installation/recommended_versions.rst (hunk 13 above)
[the remark this answers is not in the document]
Comment (greg-fer) [reply doc/nrf/installation/recommended_versions.rst:98]: Bumping question to @kotlarek-nordic and @kenr ... Could you have a look at Deidre's comment, please?

File: doc/nrf/installation/install_ncs.rst (hunk 16 above)
Comment (greg-fer) [reply doc/nrf/installation/install_ncs.rst:209]: Fixed. Should be macOS.

**Review by skogseth — APPROVED** [review]

File: doc/nrf/installation/recommended_versions.rst (hunk 13 above)
[the remark this answers is not in the document]
Comment (kotlarek-nordic) [reply doc/nrf/installation/recommended_versions.rst:98]: We provide tier definitions, which state what is guaranteed to work, here: https://developer.nordicsemi.com/nRF_Connect_SDK_dev/doc/PR-12437/nrf/installation/recommended_versions.html#requirements. Each platform (OS version + architecture) has its own tier assigned. If we need to support other ones, then, I think, you should ask TPM about that. @DeidreCasey, does this answer your question?

File: doc/nrf/installation/install_ncs.rst (hunk 17 above)
Comment (greg-fer) [reply doc/nrf/installation/install_ncs.rst]: Rephrased.

File: doc/nrf/installation/install_ncs.rst (hunk 18 above)
Comment (greg-fer) [reply doc/nrf/installation/install_ncs.rst]: Rephrased.

File: doc/nrf/installation/install_ncs.rst [hunk 20]
```diff
@@ -0,0 +1,607 @@
...
+
+      To clone the |NCS| code, complete the following steps:
+
+      1. Open the nRF Connect extension in |VSC| by clicking its icon in the :guilabel:`Activity Bar`.
+      #. In the extension's :guilabel:`Welcome View`, click on :guilabel:`Manage SDKs`.
+         The list of actions appears in the |VSC|'s quick pick.
+      #. Click :guilabel:`Install SDK`.
+         The list of available SDK versions appears in the |VSC|'s quick pick.
+      #. Select the SDK version to install.
+         In most cases, you want the SDK version that is either matching or higher than the :ref:`Toolchain version you have chosen <gs_installing_toolchain>`.
```
Comment (bencefr) [inline doc/nrf/installation/install_ncs.rst:183]: This is not true. Mostly other way around, but not exactly.
Keep in mind that the users first choose the SDK they will work with, not the toolchain, although first they need to install the toolchain.

Something like this:
```suggestion
         Make sure that your active toolchain's _major.minor_ version is at least the same or higher than the SDK version that you intend to install.
```

**Review by DeidreCasey — APPROVED** [review]

File: doc/nrf/installation/recommended_versions.rst (hunk 13 above)
[the remark this answers is not in the document]
Comment (greg-fer) [reply doc/nrf/installation/recommended_versions.rst:98]: Can I rephrase this to something like:
```suggestion
.. note::

   Before you start setting up the |NCS| toolchain, install available updates for your operating system.
   The |NCS| tools are not supported by the older versions of the operating systems listed at the beginning of this page.
```
@kotlarek-nordic , @DeidreCasey , agree?

File: doc/nrf/installation/install_ncs.rst (hunk 20 above)
Comment (nordicjm) [reply doc/nrf/installation/install_ncs.rst:183]: I don't think that's true either, the toolchain will only be built if something changes, so NCS 2.6, 2.7, 2.908 could be released and if the toolchain doesn't change then it will still be toolchain 2.5.x

Comment (bencefr) [reply doc/nrf/installation/install_ncs.rst:183]: In which case the only suggestion we can give is to install the latest and greatest and if something doesn't work in the future, then upgrade.

Comment (greg-fer) [reply doc/nrf/installation/install_ncs.rst]: I will include the recommendation to install the latest released version of the SDK and the toolchain. I think considerations about discrepancies in the versioning should go somewhere else, potentially in Updating tools and repositories (that is, separate PR).

**Review by bencefr — APPROVED** [review]

File: doc/nrf/installation/install_ncs.rst [hunk 21]
```diff
@@ -0,0 +1,617 @@
...
+      This file is loaded automatically when you run the above command.
+      See :ref:`Zephyr documentation about using zephyrrc files <zephyr:env_vars_zephyrrc>` for more information.
+
+   .. tab:: Linux
+
+      Navigate to the :file:`ncs` directory and run the following command in a terminal window:
+
+      .. code-block:: console
+
+         zephyr\zephyr-env.cmd
```
Comment (nordicjm) [inline doc/nrf/installation/install_ncs.rst:334]: incorrect

File: doc/nrf/installation/install_ncs.rst [hunk 22]
```diff
@@ -0,0 +1,617 @@
...
+      This file is loaded automatically when you run the above command.
+      See :ref:`Zephyr documentation about using zephyrrc files <zephyr:env_vars_zephyrrc>` for more information.
+
+   .. tab:: macOS
+
+      Navigate to the :file:`ncs` directory and run the following command in a terminal window:
+
+      .. code-block:: console
+
+         zephyr\zephyr-env.cmd
```
Comment (nordicjm) [inline doc/nrf/installation/install_ncs.rst:346]: incorrect

File: doc/nrf/installation/install_ncs.rst [hunk 23]
```diff
@@ -0,0 +1,617 @@
...
+            To build on the command line, complete the following steps:
+
+            1. With admin permissions enabled, download and install the `nRF Command Line Tools`_.
+            #. Restart the Toolchain Manager application.
+            #. Click the dropdown menu for the installed nRF Connect SDK version.
+
+               .. figure:: images/gs-assistant_tm_dropdown.png
+                  :alt: The Toolchain Manager dropdown menu for the installed nRF Connect SDK version, cropped
+
+               The Toolchain Manager dropdown menu options
```
Comment (nordicjm) [inline doc/nrf/installation/install_ncs.rst:611]: Missing indent

**Review by nordicjm — CHANGES_REQUESTED** [review]

File: doc/nrf/installation/install_ncs.rst (hunk 21 above)
Comment (greg-fer) [reply doc/nrf/installation/install_ncs.rst]: Let me hide out of shame for the copy-paste error. Thanks for spotting!

**Review by nordicjm — APPROVED** [review]

**Review by wbober — APPROVED** [review]

**Comment by greg-fer** [comment]
@carlescufi , could you check the changes?

File: doc/nrf/installation/install_ncs.rst [hunk 24]
```diff
@@ -0,0 +1,617 @@
...
+      ├─── zephyr/
+      └─── ...
+
+In this simplified structure preview, *<home>* corresponds to :file:`ncs/` and *<toolchain-installation>* and *<west-workspace>* correspond to the version names you installed.
+There are also additional directories, and the structure might change over time, for example if you later :ref:`change the state of development to a different revision <updating_repos>`.
+The full set of repositories and directories is defined in the :ref:`manifest file <zephyr:west-manifest-files>` (`see the file in the repository <west manifest file_>`_).
+
+.. _build_environment_cli:
+
+Set up the command-line build environment
```
[the remark this answers is not in the document]
Comment (carlescufi) [reply doc/nrf/installation/install_ncs.rst:306]: Note to reviewers, here's the rendered page: https://developer.nordicsemi.com/nRF_Connect_SDK_dev/doc/PR-12437/nrf/installation/install_ncs.html

File: doc/nrf/installation/install_ncs.rst [hunk 25]
```diff
@@ -0,0 +1,617 @@
...
+
+Depending on your preferred development environment, install the following tools:
+
+.. tabs::
+
+   .. group-tab:: nRF Connect for Visual Studio Code
+
+      * The latest version of |VSC| for your operating system from the `Visual Studio Code download page`_.
+      * In |VSC|, the latest version of the |nRFVSC|.
+        For installation instructions, see `How to install the extension`_.
```
Comment (carlescufi) [inline doc/nrf/installation/install_ncs.rst:43]: This is weird because it links to https://nrfconnect.github.io/vscode-nrf-connect/get_started/install.html which in turn then links to https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/installation/assistant.html. Something is not right, I think we need @trond-snekvik  here.

Comment (greg-fer) [reply doc/nrf/installation/install_ncs.rst:43]: Let me simplify this.

File: doc/nrf/installation/install_ncs.rst (hunk 24 above)
[the remark this answers is not in the document]
Comment (carlescufi) [reply doc/nrf/installation/install_ncs.rst:306]: Agreed to leave the section there.

File: doc/nrf/installation/install_ncs.rst (hunk 25 above)
Comment (greg-fer) [reply doc/nrf/installation/install_ncs.rst]: Pointed directly to VSC Marketplace. The page in the VSC docs will be rewritten once this PR is merged.

File: doc/nrf/installation/install_ncs.rst [hunk 26]
```diff
@@ -0,0 +1,616 @@
...
+.. tabs::
+
+   .. group-tab:: nRF Connect for Visual Studio Code
+
+      1. Open the nRF Connect extension in |VSC| by clicking its icon in the :guilabel:`Activity Bar`.
+      #. In the extension's :guilabel:`Welcome View`, click on :guilabel:`Manage toolchains`.
+         The list of actions appears in the |VSC|'s quick pick.
+      #. Click :guilabel:`Install Toolchain`.
+         The list of available toolchain versions appears in the |VSC|'s quick pick.
+      #. Select the toolchain version to install.
```
Comment (trond-snekvik) [inline doc/nrf/installation/install_ncs.rst:84]:
```suggestion
      #. In the Toolchain Management menu, click :guilabel:`Install Toolchain`.
      #. Select the toolchain version to install.
```

These "The list of ABCs appears in the Visual Studio Code's quick pick" sentences look a bit broken, and they're a bit redundant. I suggest we remove them.

File: doc/nrf/installation/install_ncs.rst [hunk 27]
```diff
@@ -0,0 +1,616 @@
...
+   .. group-tab:: nRF Connect for Visual Studio Code
+
+      To clone the |NCS| code, complete the following steps:
+
+      1. Open the nRF Connect extension in |VSC| by clicking its icon in the :guilabel:`Activity Bar`.
+      #. In the extension's :guilabel:`Welcome View`, click on :guilabel:`Manage SDKs`.
+         The list of actions appears in the |VSC|'s quick pick.
+      #. Click :guilabel:`Install SDK`.
+         The list of available SDK versions appears in the |VSC|'s quick pick.
+      #. Select the SDK version to install.
```
Comment (trond-snekvik) [inline doc/nrf/installation/install_ncs.rst:185]:
```suggestion
      #. In the extension's :guilabel:`Welcome View`, click on :guilabel:`Manage SDKs`.
      #. In the SDK management menu, click :guilabel:`Install SDK`.
      #. Select the SDK version to install.
```

File: doc/nrf/installation/install_ncs.rst (hunk 24 above)
[the remark this answers is not in the document]
Comment (trond-snekvik) [reply doc/nrf/installation/install_ncs.rst:305]: Could potentially add a sentence about how this is only necessary if you work in a freestanding application

**Review by trond-snekvik — COMMENTED** [review]
I just have a few nits to pick. Content wise, this looks good to me.

**Review by trond-snekvik — APPROVED** [review]
